### DISCUSSION TITLE
Umzug down is stuck

### DISCUSSION THREAD

**Discussion by chriml** [sequelize/umzug#554] [Q&A]
Hi all,

I am trying to reseed my database from within a cypress task. 
Initalizing and adding modules works just as normal but calling the down method seems to not do anything. Also no error message is logged.

Reseeding with the exact same process and code from the backend diretly seems to be no issue.

Any ideas why this could be a case?

Thank you!

**Comment by mmkal** [comment]
What about `executed`? Could you share a repro?
